samples/ScreenTFT_ILI9340-ILI9341/app/BMPDraw.cpp

```diff
@@ -49,8 +49,8 @@ void bmpDraw(Adafruit_ILI9341 tft, String fileName, uint8_t x, uint8_t y)
 	uint32_t rowSize;					// Not always = bmpWidth; may have padding
 	uint8_t sdbuffer[3 * BUFFPIXEL];	// pixel buffer (R+G+B per pixel)
 	uint8_t buffidx = sizeof(sdbuffer); // Current position in sdbuffer
-	boolean goodBmp = false;			// Set to true on valid header parse
-	boolean flip = true;				// BMP is stored bottom-to-top
+	bool goodBmp = false;				// Set to true on valid header parse
```

Comment:
The newly added training spaces are probably not needed here.

Reply:
Checked, there  aren't any - it's tabs added by clang-format

Reply:
Ok, then it is fine.